src/widgets/activity-group/ui/ActivitySectionList.tsx

```diff
@@ -48,6 +50,10 @@ function ActivitySectionList({ appletId, groups }: Props) {
     entityType: EntityType,
     eventId: string,
   ) {
+    if (!isFocused) {
```

Comment:
It looks weird. We're changing progress state, and make all other multiple checks, but do not do navigation. Why?